css/component_styles/alert.css

```diff
@@ -0,0 +1,64 @@
+/* -------- alert css -------- */
+:root {
+    --color-success: #38A169;
```

Comment:
It will be good to define all your variable in your main style file and link that file to the component HTML page

Reply:
done that 🛠️, thanks